src/ralph/assets/tests/factories.py

```diff
@@ -35,6 +35,20 @@ def next_mac(n):
     return ':'.join(map(lambda x: '%02x' % x, mac))
 
 
+def next_wwn(n):
+    wwn = [
+        0xaa,
+        0xbb,
+        0xcc,
+        0xdd,
+        0xee,
+        n >> 16 & 0xff,
+        n >> 8 & 0xff,
+        n & 0xff
+    ]
+    return ''.join(map(lambda x: '%02x' % x, wwn))
```

Comment:
It's basically a copy of `next_mac` from the same file (lines 26-35), which was written by you or @ar4s :wink: